### DISCUSSION TITLE
nuclei-templates\http\fuzzing\waf-fuzz.yaml in nuclei 2.9.4 error

### DISCUSSION THREAD

**Discussion by ltfafei** [projectdiscovery/nuclei#3716] [General]
validate test access, but run url error?

![image](https://github.com/projectdiscovery/nuclei/assets/43526141/619a1903-8ace-488f-86e9-6eb9693ce8c1)

**Comment by MetzinAround** [comment]
I'm not entirely sure, but it looks to me like the single template being used might not work against that target? 

@ehsandeep  could you help?

**Comment by ehsandeep** [comment]
@ltfafei see https://nuclei.projectdiscovery.io/nuclei/get-started/#template-exclusion, templates with `fuzz` tags are excluded from the default run, in order to override default, you need to use `-itags fuzz`. cc @MetzinAround 

```console
   -itags, -include-tags string[]  tags to be executed even if they are excluded either by default or configuration
```

**Comment by ltfafei** [comment]
> @ltfafei see https://nuclei.projectdiscovery.io/nuclei/get-started/#template-exclusion, templates with `fuzz` tags are excluded from the default run, in order to override default, you need to use `-itags fuzz`. cc @MetzinAround @ltfafei 参见 https://nuclei.projectdiscovery.io/nuclei/get-started/#template-exclusion ，带有 `fuzz` 标签的模板被排除在默认运行之外，为了覆盖默认值，需要使用 `-itags fuzz` 。cc @MetzinAround
> 
> ```
>    -itags, -include-tags string[]  tags to be executed even if they are excluded either by default or configuration
> ```

oh
